lib/node/Type.js

```diff
@@ -63,6 +63,9 @@ NodeType.isConstantFraction = function(node, allowUnaryMinus=false) {
   if (NodeType.isOperator(node, '/')) {
     return node.args.every(n => NodeType.isConstant(n, allowUnaryMinus));
   }
+  else if (allowUnaryMinus && NodeType.isUnaryMinus(node)) {
```

Comment:
This is in the check `isConstantFraction`

for context, when there are no variables in an equation, `solveConstantEquation` gets called in `solveEquation/stepThrough`

this in turn led to 

```
  let rNodeIsConstant = Node.Type.isConstantOrConstantFraction(equation.rightNode, true)
  let lNodeIsConstant = Node.Type.isConstantOrConstantFraction(equation.leftNode, true)

  if (!lNodeIsConstant || !rNodeIsConstant) {
    throw Error('Expected both nodes to be constants, instead got: ' +
                equation.ascii());
  }
```

Anywho, the issue is that `-⅓` was returning false for this check because it was a urnary-minus paired with a fraction.